test/e2e/kind.sh

```diff
@@ -19,20 +19,17 @@ set -euo pipefail
 
 source test/e2e/aliases.sh
 
-# images built for kind v0.20.0
-# https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0
+# images built for kind v0.24.0
+# https://github.com/kubernetes-sigs/kind/releases/tag/v0.24.0
 declare -A kind_images
 
-kind_images[1.28]="kindest/node:v1.28.0@sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31"
-kind_images[1.27]="kindest/node:v1.27.3@sha256:3966ac761ae0136263ffdb6cfd4db23ef8a83cba8a463690e98317add2c9ba72"
-kind_images[1.26]="kindest/node:v1.26.6@sha256:6e2d8b28a5b601defe327b98bd1c2d1930b49e5d8c512e1895099e4504007adb"
-kind_images[1.25]="kindest/node:v1.25.11@sha256:227fa11ce74ea76a0474eeefb84cb75d8dad1b08638371ecf0e86259b35be0c8"
-kind_images[1.24]="kindest/node:v1.24.15@sha256:7db4f8bea3e14b82d12e044e25e34bd53754b7f2b0e9d56df21774e6f66a70ab"
-kind_images[1.23]="kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb"
-kind_images[1.22]="kindest/node:v1.22.17@sha256:f5b2e5698c6c9d6d0adc419c0deae21a425c07d81bbf3b6a6834042f25d4fba2"
-kind_images[1.21]="kindest/node:v1.21.14@sha256:8a4e9bb3f415d2bb81629ce33ef9c76ba514c14d707f9797a01e3216376ba093"
+kind_images[1.30]="kindest/node:v1.30.4@sha256:976ea815844d5fa93be213437e3ff5754cd599b040946b5cca43ca45c2047114"
+kind_images[1.29]="kindest/node:v1.29.8@sha256:d46b7aa29567e93b27f7531d258c372e829d7224b25e3fc6ffdefed12476d3aa"
+kind_images[1.28]="kindest/node:v1.28.13@sha256:45d319897776e11167e4698f6b14938eb4d52eb381d9e3d7a9086c16c69a8110"
+kind_images[1.27]="kindest/node:v1.27.16@sha256:3fd82731af34efe19cd54ea5c25e882985bafa2c9baefe14f8deab1737d9fabe"
 
-K8S_VERSION="${K8S_VERSION:-"1.28"}"
+
+K8S_VERSION="${K8S_VERSION:-"1.29"}"
```

Comment:
Why default to version 1.29 and not 1.30?

Reply:
no real reason, just did latest in test matrix - 1. I've switched it to 1.30.